### ISSUE TITLE
Qui a fait ce projet ?

### ISSUE THREAD

**Issue by Clemios** [Opiderme/PT-CV#1]
Il me reste quelques projets PhP à évaluer dont celui-ci.

Je ne sais pas qui est l'étudiant derrière ce projet.

Merci de vous faire connaitre 😄

**Comment by Opiderme** [comment]
Oui, le projet TP-CV-PHP (et non PT-CV) du github Opiderme est le mien.

Le lun. 11 nov. 2024, 14:10, Clément Fleury ***@***.***> a
écrit :

**Comment by Clemios** [comment]
Le "mien" mais qui es tu ?
Je n'ai pas "Opiderme" sur la liste des étudiants 🥲

**Comment by Opiderme** [comment]
voici mon reposit : https://github.com/Opiderme/TP-CV-PHP

Le lun. 18 nov. 2024 à 18:25, Clément Fleury ***@***.***> a
écrit :

**Comment by Opiderme** [comment]
Normalement il est en public donc ça devrait le faire sinon faite moi signe.
